modules/renter/hostdb/negotiate_test.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/NebulousLabs/Sia/modules/gateway"
 	"github.com/NebulousLabs/Sia/modules/miner"
 	"github.com/NebulousLabs/Sia/modules/transactionpool"
-	"github.com/NebulousLabs/Sia/modules/wallet"
+	modWallet "github.com/NebulousLabs/Sia/modules/wallet" // name conflicts with type
```

Comment:
yeah, the wallet dependency is just a pain in general.
maybe I should just throw out the `txnBuilder` shim code. Stubbing out a full transaction builder isn't all that bad. I don't think we actually do it anywhere.